lib/qmp.mli

```diff
@@ -35,6 +35,12 @@ type fd_info = {
   fdset_id : int;
 }
 
+(*QOM properties*)
```

Comment:
Docstrings are of the form `(** ... *)`.
I would add slightly more details like `(** QOM properties - link/to/QEMU_QOM_documentation *)`

Reply:
link/to/qemu_qom_documentaition here means need to add a link here or just comments?

Reply:
updated

Reply:
It means add the actual link to the documentation

Reply:
updated